.circleci/config.yml

```diff
@@ -0,0 +1,170 @@
+          yq write --inplace deployments/postgres.yml spec.template.spec.volumes[0].hostPath.path /workspace/shopsys/$CIRCLE_BRANCH/db
+        working_directory: project-base/k8s
+
+    - run:
+        name: Set commit hash for webserver and postgres to redeploy with volumes set
+        command: |
+          yq write --inplace deployments/postgres.yml spec.template.metadata.labels.commit $CIRCLE_SHA1
+          yq write --inplace deployments/postgres.yml spec.selector.matchLabels.commit $CIRCLE_SHA1
+          yq write --inplace deployments/webserver.yml spec.template.metadata.labels.commit $CIRCLE_SHA1
+          yq write --inplace deployments/webserver.yml spec.selector.matchLabels.commit $CIRCLE_SHA1
+        working_directory: project-base/k8s
+
+    - run:
+        name: Replace docker image in deployment definition
+        command: |
+          DOCKER_IMAGE_TAG="$(cksum ../docker/php-fpm/Dockerfile | cut -d ' ' -f1)"
+          yq write --inplace deployments/webserver.yml spec.template.spec.containers[0].image $DOCKER_USERNAME/$DOCKER_IMAGE:$DOCKER_IMAGE_TAG
+        working_directory: project-base/k8s
+
+    - run:
+        name: Create configmap manifests
+        command: |
+          kubectl create configmap nginx-configuration --from-file ../../docker/nginx/nginx.conf --dry-run --output=yaml > nginx-configuration.yml
+          kubectl create configmap php-configuration --from-file ../docker/php-fpm/php-ini-overrides.ini --dry-run --output=yaml > php-configuration.yml
+        working_directory: project-base/k8s
+
+    - run:
+        name: Setup domain a parameters files
+        command: |
+          FIRST_DOMAIN_HOSTNAME=$CIRCLE_BRANCH-circle.$DEVELOPMENT_SERVER_DOMAIN
+          SECOND_DOMAIN_HOSTNAME=2.$CIRCLE_BRANCH-circle.$DEVELOPMENT_SERVER_DOMAIN
+          # Copy files from teplates
+          cp project-base/app/config/domains_urls.yml.dist project-base/app/config/domains_urls.yml
+          cp project-base/app/config/parameters_test.yml.dist project-base/app/config/parameters_test.yml
+          cp project-base/app/config/parameters.yml.dist project-base/app/config/parameters.yml
+          # Add domains into domain file
+          yq write --inplace project-base/app/config/domains_urls.yml domains_urls[0].url http://$FIRST_DOMAIN_HOSTNAME
+          yq write --inplace project-base/app/config/domains_urls.yml domains_urls[1].url http://$SECOND_DOMAIN_HOSTNAME
+
+    - run:
+        name: Set SSH credentials
+        command: |
+          echo -e $SERVER_SSL_CERT > ~/shopsys
+          chmod 0400 ~/shopsys
+
+    - run:
+        name: Upload repository into cluster, create volumes and set correct permissions
+        command: |
+          cd .. && tar -zcvf source.tar.gz source
+          ssh -o "StrictHostKeyChecking no" -i ~/shopsys www-data@heimdall.shopsys.com mkdir -p /workspace/shopsys/$CIRCLE_BRANCH
```

Comment:
Is disabling the StrictHostKeyChecking necessary?

Reply:
Maybe, I thought that this is because of that ssh key is generated somewhere else then on circler server. So while this option is on, there would be problem with host on which should be ssh key generated.